app/views/orders/index.html.erb

```diff
@@ -1,7 +1,13 @@
 <main>
+  <%= tag.h1 "#{current_user.name}'s Orders" %>
```

Comment:
I don't think I've seen this syntax before, what's the purpose of the `tag` part?